templates/_devcontainer/nginx/nginx.conf.ts

```diff
@@ -20,15 +20,19 @@ http {
     listen 80;
 
     location / {
+      error_page 418 = @websocket;      
       proxy_redirect off;
       proxy_pass     http://iobroker:8081;
+      if ( $args ~ "sid=" ) { return 418; }      
```

Comment:
remove trailing spaces:
```suggestion
      if ( $args ~ "sid=" ) { return 418; }
```